src/ui/pages/BackupWallet/clipboardWarning.ts

```diff
@@ -6,5 +6,6 @@ export const clipboardWarning = {
       seedType === SeedType.privateKey ? 'private key' : 'recovery phrase';
     return `Did you save the ${secretName}?`;
   },
-  isWarningMessage: (value: string) => value.startsWith('Did you save'),
+  isWarningMessage: (value: string | string[]) =>
+    (Array.isArray(value) ? value.join(' ') : value).startsWith('Did you save'),
```

Comment:
why not convert to string on call site? Here: https://github.com/zeriontech/zerion-wallet-extension/pull/367/files#diff-60196fc68e991de1816378ec4a5200f343de198e8255663d94a926b3a0e6b8b9R26

Otherwise it creates unnecessary coupling